src/components/Tasks/Toolbar.tsx

```diff
@@ -20,23 +24,66 @@ import { StoreState } from '../../store';
 
 import Menu from '../../widgets/Menu';
 
+const transitionTimeout = 500;
+
+const transitionStyles = {
+  entering: { visibility: 'visible', width: '100%', overflow: 'hidden' },
+  entered: { visibility: 'visible', width: '100%' },
+  exiting: { visibility: 'visible', width: '0%', overflow: 'hidden' },
+  exited: { visibility: 'hidden', width: '0%' },
+};
+
+const useStyles = makeStyles((theme) => ({
+  button: {
+    marginRight: theme.spacing(1),
+  },
+  autocomplete: {
+    transition: `width ${transitionTimeout}ms`,
+    marginRight: theme.spacing(1),
+  },
+  input: {
+    borderRadius: theme.shape.borderRadius,
```

Comment:
All of this should probably be replaced by either the standard variant, or maybe the filled of the input box. Please don't make new styles, and instead use the styles provided by the MUI library.

Reply:
I am using filled, but I'm trying to make it look better with the toolbar. I basically have all these complex styles in order to make the design fit more with our UI. The MUI TextFields just don't look good outside of the context of like a form. And the API is pretty limited in how it allows you to styles these things.

Reply:
To give you an idea of what I'm going for. Here is what I have now:
<img width="1206" alt="Screen Shot 2020-03-25 at 2 01 44 AM" src="https://user-images.githubusercontent.com/12586676/77519544-ee1a1000-6e3c-11ea-93c7-9ec051ed81d8.png">

Here are how the MUI variants look:

**Standard**
<img width="1232" alt="Screen Shot 2020-03-25 at 2 02 50 AM" src="https://user-images.githubusercontent.com/12586676/77519634-0c800b80-6e3d-11ea-9908-7da4f9fd54d4.png">

**Filled**
<img width="1221" alt="Screen Shot 2020-03-25 at 2 03 09 AM" src="https://user-images.githubusercontent.com/12586676/77519656-19046400-6e3d-11ea-9916-6603f6b2a327.png">

**Outlined**
<img width="1216" alt="Screen Shot 2020-03-25 at 2 03 24 AM" src="https://user-images.githubusercontent.com/12586676/77519669-20c40880-6e3d-11ea-8e37-b5683d630acc.png">

To me, they all kind of don't quite look right. I guess the "outlined" variant is closest to what I want. I guess I could try starting with that and modifying the styles as minimally as possible to get something looking good.

Reply:
I think all of them are more or less the same. As in, none of them is 2x the other (than than filled that's really ugly).
I'd just stick with the standard. The thing is, it's just not worth it to do things in a non-standard way because they will break the next time we update MUI. I've been burnt before, it's just not worth it... :|

Reply:
Ok, I understand. I just wish the library gave you more options for modifications. It's a great library that can do a lot of cool things, until you want to deviate slightly from standard stuff. Then it's way too complicated.

Reply:
I think that's on purpose. They want people to follow MUI, not deviate from it.

Reply:
I guess that makes sense. Maybe I just don't like Material haha. Or at least I think as a design philosophy is too rigid. I understand it's good to have rules, but I also like when apps have their own personality.

Either way, it makes development soooo much easier and faster, which I appreciate.

Reply:
I agree about the personality, but as you said, there's also a plus-side. Consider MUI as a fast iteration framework for now. Maybe in the future we can have a cohesive design for the app that deviates from MUI. :)